docs/_static/css/custom-theme.css

```diff
@@ -249,6 +145,43 @@ a:hover.reference.internal {
     font-family: Roboto,Arial,sans-serif;
 }
 
+.rst-content ol li {
+  font-family: Roboto,Arial,sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  color: #666669;
+  line-height: 21px;
+}
+
+
+/* Page Table of Contents */
+.rst-content .section .contents {
+  border-bottom: solid #DDDDE0 1px;
+  margin-bottom: 40px;
+}
+
+.rst-content .section .contents .topic-title {
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 18px;
+  margin-bottom: 8px;
+}
+
+.rst-content .section .contents ul li,
+.rst-content .section .contents ul li li {
```

Comment:
they don't have class for those nested `li` element?

Reply:
![image](https://user-images.githubusercontent.com/54555734/75493471-d96b6900-59b1-11ea-99f5-684f9f54c2b3.png)

unfortunately not, even their out-of-box css has nested li styling : 
`.wy-plain-list-disc li, .rst-content .section ul li, .rst-content .toctree-wrapper ul li, article ul li {
    list-style: disc;
    margin-left: 24px;
}`

Reply:
uhh ok then